### ISSUE TITLE
Floating windows can't be moved nor resized

### ISSUE THREAD

**Issue by uu1101** [ch11ng/exwm#97]
It seems to happen since the keysym changes in XELB:

~~~
* 1932941 Fix a typo and silence a compilation warning
* 5c7a8a9 Add necessary shift mask for event to KEYSYM conversion
* c1d4e31 Handle missing modifier keys when converting event to keysym
~~~

**Comment by ch11ng** [comment]
Thank you.  It was caused by ch11ng/xelb@5c7a8a966765c74e90e1ee5e495788681eeb451e.  Fixed in ch11ng/xelb@9c01f690f46f174c462ec538050a9bb68d64d748.

**Comment by uu1101** [comment]
Thank you!
